### PR TITLE
Migrate components to use plain result, off marionette / backbone

### PR DESCRIPTION
**Pull request by andrewkfiedler**

### CODE REVIEW

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/association/association.collection.view.js [hunk 1]
```diff
@@ -12,21 +12,23 @@
...
-      selectionInterface: this.options.selectionInterface,
+      lazyResults: this.options.lazyResults,
       knownMetacards: this.options.knownMetacards,
-      currentMetacard: this.options.currentMetacard,
+      currentLazyResult: this.options.currentLazyResult,
     }
   },
   tagName: CustomElements.register('association-collection'),
+  initialize() {
+    console.log('hello')
```
Comment (jordanwilking) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/association/association.collection.view.js]: ✏️

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/copy-coordinates/copy-coordinates.tsx [hunk 2]
```diff
@@ -84,89 +55,78 @@ const generateClipboardHandler = (
...
             onClick={generateClipboardHandler(
               `POINT (${lon} ${lat})`,
-              context,
               closeParent
             )}
           >
-            <Text>
-              <Description>Well Known Text (WKT)</Description>
+            <div>
+              <div className="opacity-75">Well Known div (WKT)</div>
```
Comment (jordanwilking) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/copy-coordinates/copy-coordinates.tsx]: "Well known div (WKT)"

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/LazyQueryResult/LazyQueryResults.tsx [hunk 3]
```diff
@@ -247,6 +260,23 @@ export class LazyQueryResults {
       {} as { [key: string]: LazyQueryResult }
     )
   }
+  /**
+   * This is purely to force a rerender in scenarios where we update result values and want to update views without resorting
+   * (resorting wouldn't make sense to do client side since there could be more results on the server)
+   * It also would be weird since things in tables or lists might jump around while the user is working with them.
+   */
+  _fakeResort() {
```
Comment (jordanwilking) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/LazyQueryResult/LazyQueryResults.tsx:268]: 😂 image

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx [hunk 4]
```diff
@@ -300,12 +299,14 @@ export const Editor = ({
...
                     case 'GEOMETRY':
                       return (
-                        <Geometry
-                          label={label}
+                        <LocationInputReact
                           onChange={(location: any) => {
-                            location === null || location === 'INVALID'
-                              ? setMode(Mode.BadInput)
-                              : setMode(Mode.Normal)
+                            console.log(location)
```
Comment (jordanwilking) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: ✏️

**Review by jordanwilking — APPROVED** [review]
LGTM

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by jordanwilking** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by jordanwilking** [comment]
✔️ Hero Successful ✔️ 

#### Steps

Note: had to do a smoke test for this one due to size of changes.

1. Verified uploading still works
2. Checked search route (basic searching, inspector tabs, visualizations, gazetteer)
3. Creating and opening searches
4. Browse and about route
5. Restore route. Verified that I could restore a "trashed" search.
6. Verified easy-to-test user settings

**Pull request merged**